packages/components/src/Button/Button.test.tsx

```diff
@@ -163,12 +163,12 @@ it("renders a Button with an icon", () => {
       type="button"
     >
       <svg
-        className="icon base"
+        className="ZDlJU6tECg7ouG-b27gya _3ctOeOMP7zLhU37n25xoZC"
```

Comment:
Are these hashes deterministic? This seems like it could be brittle...

Reply:
I wonder if there's a way to turn off the classname encoding for the `test` environment, just to avoid this mess/confusion...

Reply:
In the [rollup setup](https://github.com/GetJobber/atlantis/blob/34420a8ac694fd719fe9a7a2dd1653689ef9e901/packages/design/rollup.config.js#L15) for the `design` package, we use `[hash:base64]` for scope names which helps us create unique names that won't collide but they also look ugly. My understanding is that the generated class names stay they same, so they won't be brittle.

We could use a different function other than `[hash:base64]` to generate the class names but it won't be similar to how we generate class names for the [`components` package](https://github.com/GetJobber/atlantis/blob/367969a369682fa1a8a44dd3c4b719f50410415c/packages/components/rollup.config.js#L16).

Since `design` is an npm dependency of the components package, I don't think we can turn off the classname encoding in this case. Those classnames are generated during the build time of the package, which means when we npm install that dependency, they're already there.

@seanhealy do you have any suggestions?

Reply:
Ok, if these are deterministic, that takes my concern out.